build.gradle

```diff
@@ -42,7 +43,7 @@ gradlePlugin {
         commonCustomUserData {
             id = 'com.gradle.common-custom-user-data-gradle-plugin'
             displayName = 'Gradle Enterprise Common Custom User Data Gradle Plugin'
-            description = 'A Gradle plugin to capture common custom user data used for Gradle Build Scans in Gradle Enterprise'
+            description = releaseNotes().get()
```

Comment:
Is `description` not of type Provider<String> but a String?

Reply:
I assume the answer is NO.

Reply:
That's correct - sorry, I did verify that it is in fact a string, but forgot to respond to your comment.